torrt/trackers/anilibria.py

```diff
@@ -91,6 +91,8 @@ def find_available_qualities(self, url: str) -> Dict[str, str]:
         for torrent in torrents:
             if REGEX_RANGE.match(torrent['series']):  # filter out single-file torrents like trailers,...
                 series2torrents[torrent['series']].append(torrent)
+            elif torrent['series'] == json['data']['series']:  # ongoing, only the first episode available
```

Comment:
Inner clause is the same, why we really need `elif` instead of `or`?

Reply:
Done.